axelar-chains-config/src/utils/verifyContract.js

```diff
@@ -27,9 +27,26 @@ const verifyContract = (env, chain, contract, args, options = {}) => {
     console.log(`Verifying contract ${contract} with args '${stringArgs.join(',')}'`);
     console.log(cmd);
 
-    execSync(cmd, { stdio: 'inherit' });
```

Comment:
pipe stderr to another place, and use a try/catch to read this stderr output and avoid re-raising the error in that case?